airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -64,22 +70,34 @@ def build_refresh_request_body(self) -> Mapping[str, Any]:
 
         return payload
 
+    @backoff.on_exception(
```

Comment:
Just out of curiosity. Is it possible to disable or configure this behavior if needed? Not urgent/blocking.

Reply:
Well disabling may be a bit more more challenging than configuring it but still I think both are possible. Anyway, I think it's worth doing in a dedicated PR if we do need these options